#28 add addOnOpenListener for open-callback events (#30)
hCaptcha/hcaptcha-android-sdk: README.md

If `verifyWithHCaptcha` is called with different arguments than `setup` the SDK will handle this by re-configuring hCaptcha. Note that this will reduce some of the performance benefit of using `setup`.

> Additionally the SDK provides a listener to track hCaptcha open events, e.g. for analytics:

Reviewer: ```suggestion
The SDK also provides a listener to track hCaptcha open events, e.g. for analytics:
```

> Additionally SDK provides a listener to track hCaptcha opening (which can be used for analytics for instance), like this:

Reviewer: ```suggestion
Additionally the SDK provides a listener to track hCaptcha open events, e.g. for analytics:
```
